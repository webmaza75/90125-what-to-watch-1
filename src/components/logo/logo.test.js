import React from 'react';
import renderer from 'react-test-renderer';

import Logo from './logo.jsx';

it(`Logo correctly renders`, () => {
  const logo = renderer
    .create(<Logo />)
    .toJSON();
  expect(logo).toMatchSnapshot();
});

it(`Light Logo correctly renders (footer)`, () => {
  const logo = renderer
    .create(<Logo
      invert={true}
    />)
    .toJSON();
  expect(logo).toMatchSnapshot();
});

