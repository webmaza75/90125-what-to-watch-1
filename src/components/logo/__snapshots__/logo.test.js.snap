// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Light Logo correctly renders (footer) 1`] = `
<div
  className="logo"
>
  <a
    className="logo__link logo__link--light"
    href="main.html"
  >
    <span
      className="logo__letter logo__letter--1"
    >
      W
    </span>
    <span
      className="logo__letter logo__letter--2"
    >
      T
    </span>
    <span
      className="logo__letter logo__letter--3"
    >
      W
    </span>
  </a>
</div>
`;

exports[`Logo correctly renders 1`] = `
<div
  className="logo"
>
  <a
    className="logo__link "
    href="main.html"
  >
    <span
      className="logo__letter logo__letter--1"
    >
      W
    </span>
    <span
      className="logo__letter logo__letter--2"
    >
      T
    </span>
    <span
      className="logo__letter logo__letter--3"
    >
      W
    </span>
  </a>
</div>
`;
